middleware/github.js

```diff
@@ -20,73 +20,42 @@ const options = {
  */
 module.exports = asyncMiddleware(async (req, res, next) => {
   const authHeader = req.get('authorization');
-  if (!authHeader || !authHeader.startsWith('Bearer ')) {
+  // if there is an auth header, it had better be Bearer
+  if (authHeader && !authHeader.startsWith('Bearer ')) {
     res.status(401).end();
     return;
   }
-  const token = authHeader.split(' ')[1];
 
+  const token = authHeader ? authHeader.split(' ')[1] : null;
```

Comment:
May want to stick in a `req.hostname === 'localhost' || req.hostname === '127.0.0.1'`